libs/platform/ebpf_epoch.c

```diff
@@ -236,19 +239,16 @@ ebpf_epoch_initiate()
 
     for (cpu_id = 0; cpu_id < _ebpf_epoch_cpu_count; cpu_id++) {
         ebpf_epoch_cpu_entry_t* cpu_entry = &_ebpf_epoch_cpu_table[cpu_id];
-        cpu_entry->epoch_state.epoch = _ebpf_current_epoch;
-        cpu_entry->epoch_state.active = false;
         ebpf_lock_create(&cpu_entry->lock);
-
         ebpf_list_initialize(&cpu_entry->free_list);
     }
 
     for (cpu_id = 0; cpu_id < _ebpf_epoch_cpu_count; cpu_id++) {
```

Comment:
nit: Can we merge this `for` loop with the one at line `240`?

Reply:
No because the second for loop can exit early, which would leave the CPU state partially uninitialized, which would crash on cleanup.